civihr_default_theme/scss/pages/_onboarding-wizard.scss

```diff
@@ -722,3 +723,11 @@ $carousel-box-shadow-color: rgba(#3f3c55,0.3);
     }
   }
 }
+
+// Specific to datepicker
```

Comment:
sorry, i'm not sure i understand your explanation. I don't get it still why this needs to be done just for onboarding. What if i need to show the datepicker in some other page in the SSP? Wouldn't we end up duplicating the style then?

Reply:
> What if i need to show the datepicker in some other page in the SSP? Wouldn't we end up duplicating the style then?

Actually, i had the impression that since we are not using "custom-civihr.css" in non of the SSP pages until now. So, i thought we will be using the css for the onboarding form page only. But, i realixed that there is a chance we might need it some other SSP pages. Ad in tant case, it's not a good idea to keep the style in specific partial.

I will move it to new partial. Thanks for the suggestion.